### PR TITLE
added coffee editor subpage

### PR DESCRIPTION
**Pull request by JonasHelming**
This already contains a link to the demo, so it should be merged if all blocker issues of the demo are resolved

### CODE REVIEW

File: content/_index/coffeeeditorfeatures/diagrameditor.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++
+title = "Diagram editor"
+weight = 30
+
+[asset]
+  icon = "fas fa-project-diagram"
++++
+
+The coffee editor provides an graphical diagram editor. The example editor allows specifying the behavior of a coffee maching using a flow chart like notation. The diagram editor is based on [the graphical language server platform (Eclipse GLSP)](https://www.eclipse.org/glsp/). Double click the file "superbrewer3000.coffeenotation" in the coffee editor to try out the diagram editor!
```
Comment (eneufeld) [inline content/_index/coffeeeditorfeatures/diagrameditor.md]:
```suggestion
The coffee editor provides a graphical diagram editor. The example editor allows specifying the behavior of a coffee maching using a flow chart like notation. The diagram editor is based on [the graphical language server platform (Eclipse GLSP)](https://www.eclipse.org/glsp/). Double click the file "superbrewer3000.coffeenotation" in the coffee editor to try out the diagram editor!
```

Comment (eneufeld) [reply content/_index/coffeeeditorfeatures/diagrameditor.md]: You can also open the graphical editor from the tree editor using a right click menu on the workflow node.

File: content/_index/coffeeeditorfeatures/forms.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
++++
+title = "Tree/form-based property editor"
+weight = 20
+
+[asset]
+  icon = "fab fa-wpforms"
++++
+
+The coffee editor includes an form-based editor with a tree showing the hierarchy of the model instances. This allows to efficiently browse the model and enter data. The form editor is based on [JSON Forms](https://jsonforms.io). Double click the file "superbrewer3000.coffee" in the coffee editor to try out the editor!
```
Comment (eneufeld) [inline content/_index/coffeeeditorfeatures/forms.md]:
```suggestion
The coffee editor includes a form-based editor with a tree showing the hierarchy of the model instances. This allows to efficiently browse the model and enter data. The form editor is based on [JSON Forms](https://jsonforms.io). Double click the file "superbrewer3000.coffee" in the coffee editor to try out the editor!
```

File: content/_index/coffeeeditorfeatures/modelanalysis.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
++++
+title = "Model Analysis and Chart Visualization"
+weight = 50
+
+[asset]
+  icon = "fas fa-chart-pie"
++++
+
+The coffee editor provides an example model analysis, the result is visualized as a "sun burst" chart. The analysis is an external component written in Kotlin, the chart is based on D3. Select the file "superbrewer3000.wfconfig" in the coffee editor, press F1, type "Analyze workflow model" and hit enter to see the model analysis in action!
```
Comment (eneufeld) [inline content/_index/coffeeeditorfeatures/modelanalysis.md:9]: you can also right click inside the editor to call this action

File: content/_index/coffeeeditorlinks/sources.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Browse the sources"
+weight = 20
+
+[asset]
+  icon = "fas fa-code"
+  url = "https://github.com/eclipsesource/coffee-editor"
++++
+
+The coffee editor is an open source example modeling application. Have a look at [the sources](https://github.com/eclipsesource/coffee-editor) to find out how the features are implemented. The link above als contains instructions how to run the coffee editor yourself.
```
Comment (eneufeld) [inline content/_index/coffeeeditorlinks/sources.md]:
```suggestion
The coffee editor is an open source example modeling application. Have a look at [the sources](https://github.com/eclipsesource/coffee-editor) to find out how the features are implemented. The link above also contains instructions how to run the coffee editor yourself.
```

File: content/_index/coffeeeditorlinks/tryitnow.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Try it now!"
+weight = 10
+
+[asset]
+  icon = "fas fa-play"
+  url = "http://35.234.104.175:9091/services/launch"
```
Comment (eneufeld) [inline content/_index/coffeeeditorlinks/tryitnow.md]: @jfaltermeier can we use a different url eg eclipsesource.com/coffee-editor/try , I find the IP not very trustworthy ;-)

**Review by eneufeld — COMMENTED** [review]
There are some small typos. 
And I have a suggestion about the coffee editor try-out url.

File: content/_index/coffeeeditorfeatures/modelanalysis.md (hunk 3 above)
Comment (JonasHelming) [reply content/_index/coffeeeditorfeatures/modelanalysis.md:9]: Which editor?

Comment (eneufeld) [reply content/_index/coffeeeditorfeatures/modelanalysis.md:9]: the .wfconfig editor

Comment (JonasHelming) [reply content/_index/coffeeeditorfeatures/modelanalysis.md:9]: Hmm shall we really mention this? It breaks the layout, as the text gets longer :-)

Comment (JonasHelming) [reply content/_index/coffeeeditorfeatures/modelanalysis.md:9]: I will mention it in the welcome page

Comment (eneufeld) [reply content/_index/coffeeeditorfeatures/modelanalysis.md:9]: It was just a pointer, I don't care what you do with this information ;-)

File: content/_index/coffeeeditorlinks/tryitnow.md (hunk 5 above)
Comment (jfaltermeier) [reply content/_index/coffeeeditorlinks/tryitnow.md]: @eneufeld As a first improvement I'll create an HTTP redirect from http://coffee-editor.eclipsesource-munich.com/ 

I created it, but at least on my machine it is not propagated yet.

Comment (eneufeld) [reply content/_index/coffeeeditorlinks/tryitnow.md]: @jfaltermeier works for me, thank you.
@JonasHelming can you update the url?

Comment (JonasHelming) [reply content/_index/coffeeeditorlinks/tryitnow.md]: done

**Comment by JonasHelming** [comment]
@eneufeld can you merge this now and update the website?

**Review by eneufeld — APPROVED** [review]

**Pull request merged**